lib/identity_cache/query_api.rb

```diff
@@ -375,39 +375,23 @@ def expire_primary_index # :nodoc:
       IdentityCache.cache.delete(primary_cache_index_key)
     end
 
-    def expire_secondary_indexes # :nodoc:
-      return unless self.class.primary_cache_index_enabled
-      cache_indexes.try(:each) do |fields|
-        if self.destroyed?
-          IdentityCache.cache.delete(secondary_cache_index_key_for_previous_values(fields))
-        else
-          new_cache_index_key = secondary_cache_index_key_for_current_values(fields)
-          IdentityCache.cache.delete(new_cache_index_key)
-
-          if !was_new_record?
-            old_cache_index_key = secondary_cache_index_key_for_previous_values(fields)
-            IdentityCache.cache.delete(old_cache_index_key) unless old_cache_index_key == new_cache_index_key
-          end
-        end
-      end
-    end
-
     def expire_attribute_indexes # :nodoc:
-      cache_attributes.try(:each) do |(attribute, fields)|
+      cache_attributes.try(:each) do |(attribute, fields, unique)|
```

Comment:
Can we remove this `each`? It seems to be always an array and we are expecting it to be an array at `cache_attribute` so I think there is no point of not expecting here.